aten/src/ATen/native/native_functions.yaml

```diff
@@ -9838,6 +9838,14 @@
     CUDA: foreach_tensor_addcdiv_scalarlist_cuda_
   autogen: _foreach_addcdiv.ScalarList_out
 
+- func: _foreach_addcdiv_.Tensor(Tensor(a!)[] self, Tensor[] tensor1, Tensor[] tensor2, Tensor scalars) -> ()
```

Comment:
Curious why this is not `Tensor[] scalars`? That sounds more consistent?

Reply:
Because those are more difficult to convert to scalars efficiently.

Reply:
Well it can convert the step sizes to a torch.Tensor before calling into foreach

Reply:
Doing a .cat() before every call sounds like a big overhead?

Reply:
It's for 1-element tensors on cpu only, should be fast.